command/repo.go

```diff
@@ -406,3 +406,8 @@ func repoView(cmd *cobra.Command, args []string) error {
 	fmt.Fprintf(cmd.ErrOrStderr(), "Opening %s in your browser.\n", displayURL(openURL))
 	return utils.OpenInBrowser(openURL)
 }
+
+func slug(name string) string {
+	fields := strings.Fields(name)
+	return strings.Join(fields, "-")
```

Comment:
This works, but to avoid converting from a slice and back to a string, how about we replace by regex?

```go
nonWord := regexp.MustCompile(`[^\w.]+`)
return nonWord.ReplaceAllLiteralString(name, "-")
```

Reply:
Yeah, it works well.